src/components/Tool/Home/Tool.test.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { mdiAlert } from '@mdi/js';
+import { ThemeProvider } from '@material-ui/core';
+import theme from '../../../../src/theme';
+import TestRouter from '../../../../__tests__/util/TestRouter';
+import Tool from './Tool';
+
+const accountListId = 'account-list-1';
+
+const router = {
+  query: { accountListId },
+  isReady: true,
+};
+
+describe('Tool', () => {
+  it('props', () => {
+    const { getByTestId } = render(
+      <ThemeProvider theme={theme}>
+        <TestRouter router={router}>
+          <Tool
+            tool={'Test'}
+            desc={'Test description!!!'}
+            icon={mdiAlert}
+            id={'test'}
+          />
+          ,
+        </TestRouter>
+      </ThemeProvider>,
+    );
+    expect(getByTestId('ToolNameTypography').textContent).toEqual('Test');
+    expect(getByTestId('ToolDescriptionTypography').textContent).toEqual(
+      'Test description!!!',
+    );
```

Comment:
I think it's usually best to use test ids as last resort. I mean don't sink tons of time into avoiding them if something isn't very easy to get at but there should better ways to get elements in ways that rely on what the user experiences.

Take a look at https://testing-library.com/docs/queries/about#priority and it has a bunch of suggestions.

In this case I think you could `getByText` and assert that the returned element is in the document. Or maybe there's a way to use a role and a name filter with `getByRole`.